app/Http/Controllers/RequestPurchaseOrderController.php

```
@@ -56,9 +57,12 @@ public function updateProcessingStatus(UpdatePurchaseProcessingStatusRequest $re
         $requestPurchaseOrder->update([
             'processing_status' => $newStatus,
         ]);
+        if ($newStatus === PurchaseOrderProcessingStatus::TURNED_OVER) {
+            PurchaseOrderService::createMrrFromPurchaseOrder($requestPurchaseOrder);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Http\Services\PurchaseOrderService' in method 'updateProcessingStatus'. (Clean Code Rules)

(StaticAccess)